cmake-tool/helpers/rootserver.cmake

```diff
@@ -30,7 +30,7 @@ mark_as_advanced(TLS_ROOTSERVER)
 find_file(UIMAGE_TOOL make-uimage PATHS "${CMAKE_CURRENT_LIST_DIR}" CMAKE_FIND_ROOT_PATH_BOTH)
 mark_as_advanced(UIMAGE_TOOL)
 
-if(KernelArchRiscV)
+if(KernelArchRiscV AND KernelRiscVBBL)
```

Comment:
If the KernelRiscVBBL depends on KernelArchRiscV, the first condition is probably not needed.

Reply:
> If the KernelRiscVBBL depends on KernelArchRiscV, the first condition is probably not needed.

fixed